src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -5380,16 +5377,15 @@ void UpdateSysSizing(EnergyPlusData &state, Constant::CallIndicator const CallIn
                     // sum up the heating mass flow rate for this time step
                     Real64 adjHeatFlowSeq =
                         termUnitSizing.applyTermUnitSizingHeatFlow(zoneSizing.HeatFlowSeq(TimeStepInDay), zoneSizing.HeatFlowSeqNoOA(TimeStepInDay));
```

Comment:
In this effort I am trying to understand the definition of HeatFlowSeqNoOA (and Cool*). In the defect/example file, HeatFlowSeqNoOA = 0 and I think it should be = HeatFlowSeq. These variables are handled differently between cooling and heating in updateZoneSizingEndZoneSizingCalc7 and I think that's where the problem lies.